src/libraries/tests.proj

```diff
@@ -84,6 +84,12 @@
     <ProjectExclusions Include="$(MSBuildThisFileDirectory)System.Runtime\tests\System.Globalization.Tests\Hybrid\System.Globalization.Hybrid.WASM.Tests.csproj" />
   </ItemGroup>
 
+  <!-- invariant project cannot be tested in hybrid mode; projects using functions that throw PNSE too often to block separate tests (HashCode, CompareOptions) -->
```

Comment:
`projects using functions that throw PNSE too often to block separate tests (HashCode, CompareOptions) `

- can you explain this? And is there an issue for it?

Reply:
It's not an issue, it's by design, here is the [doc](https://github.com/dotnet/runtime/blob/69bbdb8869b781f1d86434a340a6ba4b391338f0/docs/design/features/globalization-hybrid-mode.md)

Reply:
As a follow-up job, I should add the APIs from tests that I blocked in this PR to the doc (into "Affected public APIs" sections).